### PR TITLE
[output] Normalize noise

### PR DESCRIPTION
**Pull request by NicEastvillage**
Since we know what the population and workPlace noise at most can be, we might as well normalize it to the [0,1] range. It has some neat consequences for the rendering.

### CODE REVIEW

**Comment by jenrik** [comment]
What are these "neat" consequences you are talking about, please do enlighten me

**Comment by NicEastvillage** [comment]
It's much easier to map to the 0-255 range if you are coming from the 0-1 range instead of 0-??. Before it was hardcoded for the range 0-1.8 because that was the range with the default parameters.

File: perlin.py [hunk 1]
```diff
@@ -49,19 +49,22 @@ def get_perlin_noise(x: float, y: float, base: int, scale: float = 0.005, octave
...
+    :return: the value between [0:1]
     """
     x, y = get_edge_pair_centroid(edge.getShape())
-    return get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves) + (
-            1 - (distance((x, y), centre) / radius)) * centre_weight
+    noise_value = get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves)
+    gradient = (1 - (distance((x, y), centre) / radius)) * centre_weight
+    # Normalise value to [0..1] range by dividing with its max potential value
+    normalised = (noise_value + gradient) / (1 + centre_weight)
+    return normalised
```
Comment (falkecarlsen) [inline perlin.py:67]:
```suggestion
    return (noise_value + gradient) / (1 + centre_weight)
```

**Review by falkecarlsen — APPROVED** [review]
Nice, completely missed the inaccuracy of the docstring.
We should find a better name for the function, it's also used for industry and doesn't *really* return a population **number**, rather a 'noise [unit interval](https://english.stackexchange.com/questions/275734/a-word-for-a-value-between-0-and-1-inclusive)'. Maybe `get_noise_interval`?

**Comment by NicEastvillage** [comment]
I agree the name could be improved, but I am not keen own your suggestion. It should be another PR anyway.

**Pull request merged**